human_experiments/scripts/cat/image_timestamp_scripts/start_image_timestamp.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+IMAGE_TIMESTAMP_SESSION_NAME="image-timestamp"
+
+__tmux_start_image_timestamp() {
+    local experiment_path=$1
+
+    local monitoring_lion_face="$experiment_path/lion/face_images/"
+    local monitoring_lion_screen="$experiment_path/lion/screenshots/"
+    local monitoring_tiger_face="$experiment_path/tiger/face_images/"
+    local monitoring_tiger_screen="$experiment_path/tiger/screenshots/"
+    local monitoring_leopard_face="$experiment_path/leopard/face_images/"
+    local monitoring_leopard_screen="$experiment_path/leopard/screenshots/"
+
+    mkdir -p "$monitoring_lion_face" >/dev/null
+    mkdir -p "$monitoring_lion_screen" >/dev/null
+    mkdir -p "$monitoring_tiger_face" >/dev/null
+    mkdir -p "$monitoring_tiger_screen" >/dev/null
+    mkdir -p "$monitoring_leopard_face" >/dev/null
+    mkdir -p "$monitoring_leopard_screen" >/dev/null
+
+    tmux new-session -d -s $IMAGE_TIMESTAMP_SESSION_NAME -n "lion-face" "~/git/ML4AI/tomcat/human_experiments/lab_software/tomcat-images-timestamp/build/fileChecker --out 1 --path $monitoring_lion_face"
```

Comment:
There's an environment variable called SYSTEMS_DIR that contains the path to ~git/ML4AI folder. No need to change in this PR but just to let you know.

Reply:
The problem is, this script runs on CAT computer, not TOM computer. CAT will not have access to this variable.